docs/userguide.rst

```diff
@@ -315,6 +315,33 @@ and push the created image to the DockerHub image registry:
 
     $ docker push johndoe/myenv
 
+**Supporting non-root users**
+
+Running containers with root user is not secure and therefore dicouraged. If
```

Comment:
-  You can say something like "In the Docker container ecosystem, the processes run in the containers by default using the `root` user identity.  However, this may not be secure..."
 
- typo in `discouraged`